src/main/java/fr/brouillard/oss/cssfx/CSSFX.java

```diff
@@ -40,7 +43,29 @@
 
 public class CSSFX {
     // prevent multiple global starts of CSSFX
-    private static boolean isCssFXStarted = false;
+    private static boolean isGlobalCssFXStarted = false;
+
+    private static boolean isAnyCssFXStarted = false;
+
+
+    private static BooleanProperty cssFXStarted = new SimpleBooleanProperty(false);
```

Comment:
what is the purpose of keeping several states? isn't only one enough?

Reply:
isGlobalCssFXStarted prevents starting `CSSFX.start()` twice - which would end in twice as many listeners.

isAnyCssFXStarted just tells us, whether CSSFX was started in any way.

It's worth noting, that It's not really possible to start CSSFX.